networking/ovn_kubernetes_network_provider/about-ipsec-ovn.adoc

```diff
@@ -0,0 +1,20 @@
+[id="about-ipsec-ovn"]
+= IPsec encryption configuration
+include::modules/common-attributes.adoc[]
+:context: about-ipsec-ovn
+
+toc::[]
+
+With IPsec enabled, all network traffic between pods on the OVN-Kubernetes Container Network Interface (CNI) cluster network travels through an encrypted tunnel.
+
+IPsec is disabled by default.
+
+[NOTE]
+====
+IPsec encryption can be enabled only during cluster installation.
```

Comment:
Also it can't be disabled. Worth calling out.

Reply:
@markdgray Need to mention about modes like transport?

Reply:
@anuragthehatter Yes, it is worth calling out.